src/preferences/constants.h

```diff
@@ -11,6 +11,13 @@ enum class TooltipsPreference {
     TOOLTIPS_ONLY_IN_LIBRARY = 2,
 };
 
+// Settings to enable or disable the prevention to run the screensaver.
+enum class ScreenSaverPreference {
+    PREVENT_OFF = 0,
```

Comment:
Actually, to me it's the word inhibit that isn't natural. I'm using it because that's the name used in the linux part of the solution and I kept that name. 
http://dictionary.cambridge.org/dictionary/english/prevent
http://dictionary.cambridge.org/dictionary/english/prevent